interface.go

```diff
@@ -170,7 +169,6 @@ func NewInterface(ctx context.Context, c *InterfaceConfig) (*Interface, error) {
 		writers:            make([]*udp.Conn, c.routines),
 		readers:            make([]io.ReadWriteCloser, c.routines),
 		caPool:             c.caPool,
-		disconnectInvalid:  c.disconnectInvalid,
```

Comment:
Do we need `ifce.disconnectInvalid.Store(c.disconnectInvalid)` below?

Reply:
It looks like no, because we call `ifce.reloadDisconnectInvalid(c)` after calling `NewInterface` from main.go.

This is contrast to `ifce.certState` which is handled internally in this function.

I think this approach is a little awkward because the caller may not be aware they need to initialize this value.